src/app/routes/utils/regex/utils/index.js

```diff
@@ -146,7 +146,7 @@ export const getPodcastBrandRegex = services => {
 
 export const getOnDemandRadioRegex = services => {
   const serviceRegex = getServiceRegex(services);
-  return `/:service(${serviceRegex}):variant(${variantRegex})?/:serviceId(${radioMasterBrandRegex})(/programmes)?/:mediaId(${mediaIdRegex}):amp(${ampRegex})?:lite(${liteRegex})?`;
```

Comment:
I realised we also have these tests for valid route regex, including amp routes. Do you think we should also change these in these removing tests PRs @karinathomasbbc ? There are tests for this in the routes/utils/regex file.

If so, I can also do this in the OD TV PR.

Reply:
Given that we're making these changes here, it means that we must release the Belfrage PR first, and have the .amp page 404 at the source, rather than 404ing in Simorgh.

So yes, I think we should make the corresponding change in the OD TV PR for consistency (but ideally, this probably should be part of the final cleanup / removal of all AMP code e.g. the future ticket to remove embedUrl)